test/scenarios.py

```diff
@@ -4728,11 +4728,84 @@
                     action.Wait(event.RuntimeError(Error.DB_LOST)),
                     action.NoEvent(),
                     action.Stop(),
+                    action.AssertMooseEvents(
+                        [
+                            """{
+                            "type": "exception",
+                            "arbitrary_value": -1,
+                            "code": 11,
+                            "note": "Initial DB open failed, recreating",
+                            "message": "Failed to open database",
+                            "name": "Database Error"
+                        }""",
+                            """{
+                            "type": "init",
+                            "phase": "start",
+                            "result": 0,
+                            "app_version": "*",
+                            "prod": false
+                        }""",
+                            """{
+                            "type": "init",
+                            "phase": "end",
+                            "result": 0,
+                            "app_version": "*",
+                            "prod": false
+                        }""",
+                        ]
+                    ),
                 ]
             ),
         },
         dbpath="/tmp/db/26-corrupted.sqlite",
     ),
+    Scenario(
+        "scenario26-1",
+        "Test if the instance can recover when file is innacessible",
+        {
+            "ren": ActionList(
+                [
+                    action.NoEvent(),
```

Comment:
You're missing the `DropPrivileges` actions. The runners by default run as a root